docs/users/getting_started.rst

```diff
@@ -22,51 +21,27 @@ Terminator encloses all terminology in glossaries. Terminology is managed in a
 concept-centric way, allowing several terms for each concept, even for the same
 language.
 
-
-
 Export
 ======
 
 Export to TBX
 -------------
 
-Terminator is capable of exporting to a subset of the ISO 30042 TermBase
-eXchange (TBX) standard. See :ref:`tbx-conformance` for more details.
-
-
-
+Terminator can export to a subset of the ISO 30042 TermBase eXchange (TBX) standard. See :ref:`tbx-conformance` for more details.
 
 Import
 ======
 
 Import from TBX
 ---------------
 
-Terminator is capable of importing from a subset of the ISO 30042 TermBase
-eXchange (TBX) standard. See :ref:`tbx-conformance` for more details.
-
-
-
-
-.. warning:: Terminator assumes that some elements that are used in the TBX
-   file, for example the languages, are already created in Terminator. If
-   those elements don't exist in Terminator then the import will fail showing
-   an error message.
-
-.. warning:: Terminator assumes that some required elements are present in the
-   TBX file, thus if those elements are not present then the import process
-   will fail showing an error message. See :ref:`tbx-conformance` for more
-   details on the required elements.
-
-.. note:: If any other element, besides the supported by Terminator ones, is
-   present in the TBX file Terminator won't import those, but the importing
-   process won't fail. See :ref:`tbx-conformance` for more details on which are
-   the supported elements.
+Terminator can import from a subset of the ISO 30042 TermBase
+eXchange (TBX) standard. See :ref:`tbx-conformance` for details.
 
+.. warning:: Some elements in the TBX file, for example the languages, must already be created in Terminator prior to the import, otherwise the import will fail and display an error message.
 
+.. warning:: Some required elements must be present in the TBX file. And, Terminator will not import any other elements (beside the supported elements) available in the TBX file See :ref:`tbx-conformance` for details on the required and supported elements.
```

Comment:
```suggestion
.. warning:: There are required elements in a TBX file and Terminator will not import any unsupported elements. See :ref:`tbx-conformance` for details about required and supported TBX file elements. Prior to the import, some elements that are part of the TBX file must already be created in Terminator, for example, the languages. Otherwise, the import will fail and display an error message. 
```